src/components/inputs/text-input/README.md

```diff
@@ -22,17 +22,15 @@ import { TextInput } from '@commercetools-frontend/ui-kit';
 | `value`                | `string` |    ✅    | -                            | -       | Value of the input                                                                                     |
 | `autoComplete`         | `string` |    -     | -                            | -       | Used as HTML `autocomplete` property                                                                   |
 | `onChange`             | `func`   |    -     | -                            | -       | Called with the new value. Required when input is not read only. Parent should pass it back as `value` |
-| `onBlur`               | `func`   |    -     | -                            | -       | Called when field is blurred                                                                           |
-| `onFocus`              | `func`   |    -     | -                            | -       | Called when field is focused                                                                           |
-| `isAutofocussed`       | `bool`   |    -     | -                            | -       | Focus the input field on initial render                                                                |
-| `isDisabled`           | `bool`   |    -     | -                            | `false` | Indicates that the field cannot be used (e.g not authorised, or changes not saved)                     |
-| `isReadOnly`           | `bool`   |    -     | -                            | `false` | Indicates that the field is displaying read-only content                                               |
+| `autoFocus`            | `bool`   |    -     | -                            | -       | Focus the input field on initial render                                                                |
+| `disabled`             | `bool`   |    -     | -                            | `false` | Indicates that the field cannot be used (e.g not authorised, or changes not saved)                     |
```

Comment:
I'm also not sure how difficult / annoying the refactoring will be, since even if we change all of our components to use `disabled` instead of `isDisabled`, there are probably components inside the MC that use that prop and find and replace probably won't work well...

Reply:
OK, so I'm thinking of mapping the old props to the new ones and throw a warning when the old ones are used, the same way I did in #631.

I'll also investigate how to make the code mod.

Reply:
Yeah that sounds like a good idea to me, then we can slowly do the refactor in the MC

Reply:
I've added the deprecation warnings and mapped the old props to the new.

Reply:
We have a couple of old codemods in the `transforms` folder of the mc-fe repo, in case it helps.

Reply:
I've added a codemod and tested it on UI-kit (refactored TextField to use the new props).

Reply:
@emmenko @montezume 
If we're happy with this, we can start doing the same for _all_ other inputs (and buttons, and other components that reimplement native HTML attributes)